js/components/message.jsx

```diff
@@ -1,12 +1,16 @@
 var React = require('react');
+require('../../stylesheets/message.scss');
 
 var Message = React.createClass({
   render: function() {
+    var message = this.props.message;
     return (
-      <div className='message'>
-        [{this.props.message.timestamp}]
-        {this.props.message.user}:
-        {this.props.message.text}
+      <div className="message">
+        <div className="sender-stamp">
+          <div className="message-sender">{message.user}</div>
+          <div className="message-time">{this.props.displayTime}</div>
```

Comment:
If none of the message props are changed does the message still re-render? Or all all the nodes always callled and then react decides where it has to update the dom. Now that I say it out loud it is probably the latter. But the reason I did this way, I think, was worry about the former.